src/main/java/org/jabref/logic/importer/fetcher/MrDLibFetcher.java

```diff
@@ -54,13 +60,13 @@ public String getName() {
             try {
                 if (importer.isRecognizedFormat(response)) {
                     parserResult = importer.importDatabase(response);
+                    heading = importer.getRecommendationsHeading();
```

Comment:
As said earlier, I'm not a big fan that the description and header of the tab is set dynamically based on the response. But if you really think this adds a real value to the user experience than it would be fine with me. @JabRef/developers what do you think?

Reply:
We think it's beneficial to users to see how their recommendations were generated. We're open to suggestion on this from the JabRef devs.